### ISSUE TITLE
CSS and ordering issue

### ISSUE THREAD

**Issue by marcelsalathe** [crowdAI/crowdai#329]
This image is from the homepage right now:
<img width="613" alt="screen shot 2017-09-28 at 8 12 12 pm" src="https://user-images.githubusercontent.com/215057/30983001-67270442-a489-11e7-869a-2a8f3e32c334.png">

There are two problems here: the first is that the top padding of the first challenge (to the horizontal line) is too large.

The second is that the open challenge should be at the top. I assume we simply sort by start date. But we should have a two-step ordering: first, sort by running / ended status. Then, within those, sort by start date.

**Comment by scarroll32** [comment]
These are both acting as designed.

The sorting changes were requested in closed issue #230. A high level **featured challenge** field was added to the challenges setup page, to override the sort order based on status, for the landing page only. 

Options:
(1) Clear it manually for each challenge after we no longer want a challenge pinned to the top of the list. This is the current behaviour, and can be quickly applied via the challenge configuration page.
(2) Clear the sort field automatically it when the challenge is closed.

 If 2 then we will lose the ability to pin a challenge to a specific point on the landing page.

@marcelsalathe let me know which option you prefer.

It works like this:

```
  default_scope { order("featured_sequence DESC,
                         CASE status_cd
                          WHEN 'running' THEN 1
                          WHEN 'starting_soon' THEN 2
                          WHEN 'completed' THEN 3
                          WHEN 'cancelled' THEN 4
                          WHEN 'draft' THEN 5
                          ELSE 6
                        END, created_at DESC") }
```

https://github.com/crowdAI/crowdai/issues/230

The header spacing is as per the design, see screen shot below of **template-index.html**

<img width="1223" alt="screen shot 2017-09-29 at 09 23 33" src="https://user-images.githubusercontent.com/11340230/31005186-edfd24a6-a4f7-11e7-904a-cc5afae4645c.png">

**Comment by marcelsalathe** [comment]
Ok I think we should review the design issue with Jason. I did not know there was a field that can overwrite the ordering. This needs maybe some more thought, but that aspect I consider closed for now.
